packages/block-editor/src/components/content-lock/modify-content-lock-menu-item.js

```diff
@@ -12,42 +12,45 @@ import { store as blockEditorStore } from '../../store';
 import { unlock } from '../../lock-unlock';
 
 // The implementation of content locking is mainly in this file, although the mechanism
-// to stop temporarily editing as blocks when an outside block is selected is on component StopEditingAsBlocksOnOutsideSelect
+// to stop editing a content only section an outside block is selected is on component StopEditingContentOnlySectionOnOutsideSelect
```

Comment:
Tiniest of nits, feel free to ignore!

```suggestion
// to stop editing a content only section when an outside block is selected is in the component StopEditingContentOnlySectionOnOutsideSelect
```